packages/@o3r/eslint-config-otter/schematics/ng-update/v11.6/update-configs/update-configs.spec.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  Tree,
+} from '@angular-devkit/schematics';
+import {
+  updateEslintRecommended,
+} from './update-configs';
+
+describe('updateEslintRecommended', () => {
+  it('should update configs', async () => {
+    const initialTree = Tree.empty();
+    initialTree.create('random.file', 'a file containing json-recommended');
+    initialTree.create('/test/.eslintrc.json', '{ "extends": ["@o3r/json-recommended", "@other/json-recommended"] }');
+    initialTree.create('/src/eslint.config.mjs', 'export { "extends": ["plugin:@o3r/json-recommended", "@o3r/template-recommended"] };');
```

Comment:
config with mjs does not work with extends 
this package is supposed to be used only with eslintrc config 
so we don't need to validate that no ?